docs/covidcast-py/html/_sources/getting_started.rst.txt

```diff
@@ -104,6 +104,18 @@ May 1st were updated on May 23rd based on new data, giving a ``lag`` of 22 days.
 See the :py:func:`covidcast.signal` documentation for details on the returned
 data frame.
 
+.. _api-key-usage:
+.. note ::
+
+   By default, this package submits queries to the API anonymously. If you have an
+   API key, you can use it with this package by calling
+   :py:func:`covidcast.use_api_key`, then call fetch functions as normal:
```

Comment:
Worth linking the API key registration here as well? Or is it sufficient that users can click on the use_api_key() docs where it says there (which would also mean we don't have to maintain it in two places).

Reply:
in favor of minimizing number of places to maintain information